packages/amplify_api_plugin_interface/lib/src/types.dart

```diff
@@ -24,6 +25,18 @@ export 'RestAPI/RestOptions.dart';
 export 'RestAPI/RestResponse.dart';
 export 'RestAPI/RestException.dart';
 
+export 'types/models/model.dart';
```

Comment:
I would suggest adding a prominent TODO comment for all the places where we are temporarily duplicating dependencies in API so it's super obvious in the code and no chance to miss if someone were to try to merge back to main. I would suggest something like:

```TODO: Datastore dependencies temporarily duplicated in API. Eventually they should be moved to core or otherwise reconciled to avoid duplication.```

I would add this comment here, as well as in every file this applies to. One benefit of this is that we can later search the code for this comment to fix it and reduce the chances of missing a file.

Reply:
Good call out, I've added it to all files.